packages/cdktf-cli/bin/cmds/ui/models/terraform.ts

```diff
@@ -49,6 +49,10 @@ export interface TerraformOutput {
   value: string | Record<string, unknown> | Array<any>;
 }
 
+export function isTerraformOutput(output: any): output is TerraformOutput {
+  return typeof output === "object" && typeof output.value === "string";
```

Comment:
But `output.value` is still allowed to be an array which would not return true here?